vehicle/mkvi/hardware/nostril/dashboard/BUILD

```diff
@@ -0,0 +1,5 @@
+load("//bazel:defs.bzl", "kicad_hardware")
+
+kicad_hardware(
+    name = "template_board",
```

Comment:
Can you update the BUILD file to match your project (ask Amit for examples) so that this passes CI properly :)